### PR TITLE
Add grafana and private code section to OF Pro overview.

### PR DESCRIPTION
**Pull request by welteki**
Signed-off-by: Han Verstraete (OpenFaaS Ltd) <han@openfaas.com>

## Description

- Add a section to OpenFaaS Pro that shows the Grafana dashboards we provide to Pro customers.
- Add a section that mentions the templates and faas-cli plugin to build functions that need private code.

## Motivation and Context

Show additional value of OpenFaaS Pro

## How Has This Been Tested?

Verified site locally.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`

### CODE REVIEW

File: docs/openfaas-pro/introduction.md [hunk 1]
```diff
@@ -58,6 +58,63 @@ Build functions at scale - for services providers and large teams:
 
 * [Build functions via REST API](/openfaas-pro/builder) using source code without the need to create and manage dozens or hundreds of independent CI jobs.
 
+### Grafana dashboards
```
Comment (alexellis) [inline docs/openfaas-pro/introduction.md:61]: I was thinking these would be broken out into separate sections in the ToC

File: docs/openfaas-pro/introduction.md [hunk 2]
```diff
@@ -58,6 +58,63 @@ Build functions at scale - for services providers and large teams:
...
+
+OpenFaaS comes with [built-in Prometheus metrics](/architecture/metrics). We provide a collection of Grafana dashboards to customers to help with monitoring OpenFaaS and individual functions.
+
+Available dashboards:
+
+- Autoscaling and usage metrics dashboard - Get an overview of all deployed functions and metrics for the OpenFaaS gateway.
+- Spotlight dashboard - Get detailed CPU/RAM and Rate, Error, Duration (RED) metrics for individual functions. 
+- Queue worker dashboard - Monitor the work queue for your async function invocations.
+- Builder dashboard - Dashboard for monitoring the OpenFaaS [function builder API](/openfaas-pro/builder).
```
Comment (alexellis) [inline docs/openfaas-pro/introduction.md:70]: Function Builder API

File: docs/openfaas-pro/introduction.md (hunk 1 above)
Comment (welteki) [reply docs/openfaas-pro/introduction.md:61]: Created a sub section for each dashboard.

**Review by alexellis — APPROVED** [review]
Approved. Thank you

**Comment by alexellis** [comment]
I'm just picking up on the image URLs now, since these are generated from a private repo, I would like you to save each file and upload them as part of the docs website. The URLs may not be stable, and could be removed / invalidated.

Thanks

**Pull request merged**